xDrip/Services/CloudSync/NightscoutService.swift

```diff
@@ -32,19 +35,40 @@ final class NightscoutService {
         if let settings = User.current.settings.nightscoutSync, settings.isFollowerAuthed {
             startFetchingFollowerData()
         }
+        
+        let onUpdate: (ReachabilityService) -> Void = { [weak self] reachability in
+            self?.currentConnectType = reachability.connection
```

Comment:
Still the same - when wifi connection becomes available, you're not trying to do the upload.